lib/saml/settings_service.rb

```diff
@@ -102,6 +102,8 @@ def settings
         settings.assertion_consumer_service_url = Settings.saml.callback_url
         settings.certificate_new = Settings.saml.certificate_new
 
+        settings.name_identifier_format = 'urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified'
```

Comment:
See thread here: https://dsva.slack.com/archives/C6WGBD0SW/p1531510581000271?thread_ts=1531510275.000205&cid=C6WGBD0SW

Tanel recommended setting it to `persistent` instead of `unspecified` (setting it to `unspecified` was causing id.me to fall back to `persistent`)